flink-runtime/src/main/java/org/apache/flink/runtime/util/FatalExitExceptionHandler.java

```diff
@@ -32,6 +32,7 @@
 	private static final Logger LOG = LoggerFactory.getLogger(FatalExitExceptionHandler.class);
 
 	public static final FatalExitExceptionHandler INSTANCE = new FatalExitExceptionHandler();
+	public static final int EXIT_CODE = -17;
```

Comment:
Since you are touching this already: I would propose to use a different exit code for the fatal exit code handler than here: https://github.com/apache/flink/blob/master/flink-runtime/src/main/java/org/apache/flink/runtime/util/JvmShutdownSafeguard.java#L44

Reply:
I would see this an orthogonal change.

Reply:
I filed a ticket to track this: https://issues.apache.org/jira/browse/FLINK-20425